zipline/finance/performance/tracker.py

```diff
@@ -296,6 +296,8 @@ def process_trade(self, event):
                 perf_period.handle_cash_payment(cash_adjustment)
 
     def process_transaction(self, event):
+        # Transaction events also count as trade events for the trackers
+        self.process_trade(event)
```

Comment:
Something like:

```
    def _update_last_sale(self, event):
        # updates last sale, and pay out a cash adjustment
        cash_adjustment = self.position_tracker.update_last_sale(event)
        if cash_adjustment != 0:
            for perf_period in self.perf_periods:
                perf_period.handle_cash_payment(cash_adjustment)

    def process_trade(self, event):
        self._update_last_sale(event)

    def process_transaction(self, event):
        self._update_last_sale(event)
        self.txn_count += 1
        self.position_tracker.execute_transaction(event)
        for perf_period in self.perf_periods:
            perf_period.handle_execution(event)
```
? Would this create an unnecessary attribute access on process_trade that could impact perf (or is that negligible)?